spec/lib/local-links-manager/import/local_authorities_url_importer_spec.rb

```diff
@@ -50,5 +50,51 @@ def stub_slug_csv_request(data)
       la_allerdale = LocalAuthority.find_by(snac: "16UB")
       expect(la_allerdale.homepage_url).to eq("https://www.allerdale.gov.uk")
     end
+
+    context "when any homepage URLs are blank or empty after an import" do
+      it "should alert Icinga" do
+        FactoryGirl.create(:local_authority, name: 'adur', snac: "45UB", homepage_url: nil, gss: "E07000223")
+        FactoryGirl.create(:local_authority, name: 'allerdale', snac: "16UB", homepage_url: nil, gss: "E07000026")
+        FactoryGirl.create(:local_authority, name: 'hogsmeade', snac: "WZRD", homepage_url: nil, gss: "M491C0000")
```

Comment:
Now that we're using sequences in our LocalAuthority factory, would it be worth simplifying the test setup here by not specifying things that we don't have to any more (e.g. the gss code)?